keyboards/polykybd/tools/status_oled_preview.py

```
@@ -195,6 +195,55 @@ def build_fw_confirm_panel(side, small):
+    """Settings -> "More" telemetry screen — mirror of oled_helper.c's
+    oled_telemetry_screen(). Four lines on the 64px panel (two on a 32px one),
+    each centred in its own band from its own bbox, exactly as the FW-2 confirm
+    screen above and the C do.
+
+    `usb_side` picks the half: the USB (master) half is the only one that owns the
+    split-link counters, so the LINK line reads "n/a" on the other — the C refuses
+    to render the slave's all-zero counters as a perfect 0.0%. `link` is
+    (err_permille, frames) for the master, or None for a link that has sent
+    nothing yet."""
+    pts = []
+    setp = lambda px, py: pts.append((px, py))
```

Ruff (0.16.2)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `setp` as a `def`

(E731)